.github/workflows/tfenv-updatecli.yaml

```diff
@@ -0,0 +1,41 @@
+name: updatecli-tfenv
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Run once a week
+    - cron: '0 2 * * 1'  # Every monday at 2am UTC
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  updatecli:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Updatecli in the runner
+        uses: updatecli/updatecli-action@v2
+        with:
+          version: v0.33.3
+
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.DEPENDENCY_UPDATER_APP_ID }}
+          private_key: ${{ secrets.DEPENDENCY_UPDATER_APP_PRIVATE_KEY }}
+
+      - name: Run Updatecli in Dry Run mode
+        run: updatecli diff --config ./updatecli/tfenv-weekly.d --values ./updatecli/values.github-action.yaml
```

Comment:
do we need a different action for each? I thought updatecli would take a directory of files and process them all creating a PR for each

e.g. see how it's used here:
https://github.com/jenkins-infra/jenkins-infra/blob/b8493cb24d9e6f68be8cc8dcee5f9fc260526a83/.github/workflows/updatecli.yaml#L27

https://github.com/jenkins-infra/jenkins-infra/tree/production/updatecli/weekly.d

Reply:
You are correct doesn't look like we need workflow action for each config.  I have made changes so only one workflow is present.